workflows/Snakefile

```diff
@@ -1,321 +1,13 @@
 """Snakemake file defining the workflows"""
 
-import json
-import logging
 
-import pyggdrasil as yg
+################################################################################
+# Imports Utilities
+include: "tree_inference.smk"
+include: "analyze.smk"
+include: "visualize.smk"
+# Import Experiments
+include: "mark00.smk"
+################################################################################
```

Comment:
Structured the snakemake workflows now alike our modules and experiments. The main snakemake is just to bind them all together.

Reply:
Very, very nice!